optaplanner-core/src/test/java/org/optaplanner/core/api/score/stream/bi/BiConstraintStreamTest.java

```diff
@@ -53,6 +54,11 @@ public BiConstraintStreamTest(boolean constraintMatchEnabled, ConstraintStreamIm
     // Filter
     // ************************************************************************
 
+    @Before
+    public void assumeBavet() {
+        super.assumeBavet(); // Skip tests if we're not using Bavet
```

Comment:
Nice to have:
the superclasses method and this method have the same name, but they are different:
- one is the check if it's bavet
- the other actives that check for all tests in this subclass

Maybe we can rename the second method for disableAllTestsForDrools (or a better name)?